src/page-response-impact/hooks/useSystemWideData.tsx

```diff
@@ -58,8 +60,12 @@ export function useSystemWideData(
       modelInputs,
     );
 
+    // Default Original Date 12/31/2019
```

Comment:
I think people will look at these two lines in succession and assume there's an off by one error. Can you comment why this is correct?